tool/travis.sh

```diff
@@ -23,6 +23,18 @@ tuneup check
 # Ensure we can build the app.
 pub run webdev build
 
+# Verify that the libraries are error and warning-free.
```

Comment:
This is handled in the `tuneup` step above -

Reply:
ah yes. removed.